filters/ratelimit/ratelimit.go

```diff
@@ -248,7 +249,7 @@ func clusterRatelimitFilter(maxShards int, args []interface{}) (*filter, error)
 	if keyShards > 1 {
 		f.settings = ratelimit.Settings{
 			Type:       ratelimit.ClusterServiceRatelimit,
-			Group:      group,
+			Group:      group + "." + strconv.Itoa(maxShards),
```

Comment:
Should it be `keyShards` for consistency with `MaxHits`?

Reply:
fixed